lms/templates/ux/reference/teams-create.html

```diff
@@ -0,0 +1,200 @@
+    <section class="course-content" id="course-content">
+
+    <header class="page-header">
+      <div class="header-main">
+        <nav class="breadcrumbs">
+          <a class="nav-item" href="">Solar Energy</a> <span class="icon fa-angle-right" aria-hidden="true"></span>
+        </nav>
+        <h2 class="page-title">Create a New Team</h2>
+        <p class="page-description">If you cannot find an existing team to join or would like to team up with a group of friends, create a new team.</p>
+      </div>
+    </header>
+
+    <div class="page-content">
+
+      <form class="create-team">
+      <fieldset class="required-wrapper">
+          <legend>Required Information</legend>
+
+          <div class="form-field required">
+            <label for="team-name" class="label">Team Name (Required)</label>
+            <input id="team-name" aria-required="true" class="form-input" />
+            <span class="tip tip-input">The name that will identify your team</span>
+          </div>
+
+          <div class="form-field required has-errors">
+            <label for="team-description" class="label">Team Description (Required)</label>
+            <textarea id="team-description" aria-required="true" class="form-input"></textarea>
+            <span class="tip tip-input">A short description of the team to help other students understand the goals or directives the team is pursuing</span>
+          </div>
+        </fieldset>
+        <fieldset class="optional-wrapper">
+          <legend>Optional Characteristics</legend>
+          <p class="description">Help other students find and join your team by specifying characteristics. The more limitations you add, the fewer students may be interested in joining, so choose carefully.</p>
+          <div class="form-field">
+            <label for="team-language" class="label">Language</label>
+            <select id="team-language" name="team-language" class="form-select">
+              <option value="">Select...</option>
+              <option>Swahili</option>
+              <option>Mandarin</option>
+              <option>English</option>
+              <option>Hobbitish</option>
+            </select>
+            <span class="tip tip-input">The primary language of the team</span>
+          </div>
+          <div class="form-field">
+            <label for="team-country" class="label">Country</label>
+            <select id="team-country" name="team-country" class="form-select">
+              <option value="">Select...</option>
+              <option>Timbuktoo</option>
+              <option>The Moon</option>
+              <option>Paris</option>
+              <option>JP</option>
+            </select>
+            <span class="tip tip-input">The primary country of the team</span>
+          </div>
+        </fieldset>
+
+        <div class="form-actions">
+          <button class="action action-primary">Create</button>
+          <a class="action action-secondary" href="">Cancel</a>
```

Comment:
Hmm, channeling my inner @clrux on this one, I think we want this element to be a button since it "does something". We are trying to use links only when it brings a user "to something". Mind syncing up?